packages/perseus/src/util/parse-perseus-json.test.ts

```diff
@@ -0,0 +1,10 @@
+import parsePerseusJSON from "./parse-perseus-json";
+
+describe("parsePerseusJSON", () => {
+    it("should parse JSON", () => {
+        const result = parsePerseusJSON(
+            `{ "question": { "content": "idk why I'm testing this" }}`,
+        );
+        expect(result.question.content).toBe("idk why I'm testing this");
```

Comment:
Well, at a minimum we're validating that Typescript isn't complaining about types here. 🤷‍♂️